pages/spaces.jsx

```diff
@@ -148,7 +148,7 @@ var SessionsPage = React.createClass({
 
         <div className="white-background">
           <div className="content centered wide">
-            <h1 className="mt-5">Experiences</h1>
+            <h1 className="mt-5">2017 Experiences</h1>
```

Comment:
Changes in this page fixes https://github.com/mozilla/mozillafestival.org/issues/822